bedrock/firefox/templates/firefox/os/faq.html

```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
+                            {{_('We work with a worldwide community to create open source products like Mozilla Firefox and to innovate for the benefit of the individual and the betterment of the Web.')}}
+                            {{_('The result is great products built by passionate people and better choices for everyone.')}}
+                        </p>
+                    {% endif %}
+                    </dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Buying Firefox OS')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>{{_('What phones will Firefox OS be available on?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {{_('Since launching, Firefox OS smartphones are available in over 40 markets from multiple global operators on a number of different devices, across different price device categories. These include handsets from Alcatel ONETOUCH, ZTE, Huawei and LG.')}}
+                    {% else %} 
+                        {% trans link=url('firefox.os.devices') %}
+                        To see a full list of available Firefox OS phones, please visit the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('Where can I buy Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {% trans link=url('firefox.os.devices') %}
+                        The full list of where to buy Firefox OS devices can be found at the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% else %} 
+                        {% trans link=url('firefox.os.devices') %}
+                        To purchase a Firefox OS phone, please visit the <a href="{{ link }}">devices page</a> and select your available country.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('What will be the price of the phone?')}}</dt>
+                    <dd>{{_('Pricing is not set by Mozilla and will be established by our operator and manufacturer partners based on local market dynamics.')}}</dd>
+
+                    <dt>{{_('What apps will be available on Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        <p>{{_('The Firefox Marketplace offers an open app environment that leverages the scale of the Web as a development platform; allowing operators, phone manufacturers and developers greater flexibility.')}}</p>
+                        <p>
+                            {% trans url='https://marketplace.firefox.com/' %}
```

Comment:
I believe the reason why there are both "link" and "url" variable names is that, link points to a page in mozilla.org while url points to an external page. Will keep as is for the sake of flod's comments below.